envi/cli.py

```diff
@@ -476,6 +478,17 @@ def do_script(self, line):
         locals = self.getExpressionLocals()
         locals['argv'] = argv
 
+        if line.startswith("?"):
```

Comment:
This should still use argv since it's possible for a real filename to actually begin with the "?" char